server/db/migrations/19_create_host_node_stats_created_at_index.rb

```diff
@@ -0,0 +1,5 @@
+class CreateHostNodeStatsCreatedAtIndex < Mongodb::Migration
+  def self.up
+    HostNodeStat.create_indexes
```

Comment:
maybe migration numbering should be changed to something like "201703081345_foofoo.rb"